converters/fromZigbee.js

```diff
@@ -3480,14 +3484,84 @@ const converters = {
             }
+            if (msg.data.hasOwnProperty('1025')) {
+                result.main_cycle_output = cycleOutputLookup[msg.data['1025']];
+            }
             if (msg.data.hasOwnProperty('1026')) {
-                result.backlight_auto_dim = lookup1[msg.data['1026']];
+                const lookup = {0: 'on_demand', 1: 'sensing'};
+                result.backlight_auto_dim = lookup[msg.data['1026']];
+            }
+            if (msg.data.hasOwnProperty('1028')) {
+                result.aux_cycle_output = cycleOutputLookup[msg.data['1028']];
+            }
+            return result;
+        },
+    },
+    sinope_TH1400ZB_specific: {
+        cluster: 'manuSpecificSinope',
+        type: ['readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const result = {};
+            if (msg.data.hasOwnProperty('outdoorTempToDisplay')) {
+                result.outdoor_temp_to_display = msg.data['outdoorTempToDisplay'];
+            }
+            if (msg.data.hasOwnProperty('outdoorTempToDisplayTimeout')) {
+                result.enable_outdoor_temperature = msg.data['outdoorTempToDisplayTimeout'] == 10800 ? 'on' : 'off';
+            }
+            if (msg.data.hasOwnProperty('currentTimeToDisplay')) {
+                result.currentTimeToDisplay = msg.data['currentTimeToDisplay'];
+            }
+            if (msg.data.hasOwnProperty('floorControlMode')) {
+                const lookup = {1: 'ambiant', 2: 'floor'};
+                result.floor_control_mode = lookup[msg.data['floorControlMode']];
+            }
+            if (msg.data.hasOwnProperty('ambiantMaxHeatSetpointLimit')) {
+                result.ambiant_max_heat_setpoint = msg.data['ambiantMaxHeatSetpointLimit'] / 100.0;
+                if (result.ambiant_max_heat_setpoint === -327.68) {
+                    result.ambiant_max_heat_setpoint = 'off';
+                }
+            }
+            if (msg.data.hasOwnProperty('floorMinHeatSetpointLimit')) {
+                result.floor_min_heat_setpoint = msg.data['floorMinHeatSetpointLimit'] / 100.0;
+                if (result.floor_min_heat_setpoint === -327.68) {
+                    result.floor_min_heat_setpoint = 'off';
+                }
+            }
+            if (msg.data.hasOwnProperty('floorMaxHeatSetpointLimit')) {
+                result.floor_max_heat_setpoint = msg.data['floorMaxHeatSetpointLimit'] / 100.0;
+                if (result.floor_max_heat_setpoint === -327.68) {
+                    result.floor_max_heat_setpoint = 'off';
+                }
+            }
+            if (msg.data.hasOwnProperty('temperatureSensor')) {
+                const lookup = {0: '10k', 1: '12k'};
+                result.floor_temperature_sensor = lookup[msg.data['temperatureSensor']];
+            }
+            if (msg.data.hasOwnProperty('floorLimitStatus')) {
+                result.floorLimitStatus = msg.data['floorLimitStatus'];
```

Comment:
`floorLimitStatus` -> `floor_limit_status`

Reply:
Attribute removed. I forgot to remove it.